src/OpenLoco/src/Intro.cpp

```diff
@@ -25,6 +34,32 @@ namespace OpenLoco::Intro
     // 0x0046AE0C
     void update()
     {
+        addr<0x0005252E0, int32_t>() = 1;
+        if (_state == State::end)
+        {
+            Gfx::getDrawingEngine().getDrawingContext().clearSingle(Gfx::getScreenRT(), 0x0A);
+            if (!_50C196)
+            {
+                // Audio::stopIntro(); TODO: I think we might have broken audio in the intro...
```

Comment:
I don't think there was ever any audio in Locomotion's intro... Unlike RCT2, Locomotion just refers to the EULA until clicked.

Reply:
yeah just realised that. Looks like this was the old RCT2 code.